ds/data/processed/eurostat/rd_government_eur_hab.nuts1.csv

```diff
@@ -0,0 +1,438 @@
+year,nuts_id,nuts_year_spec,rd_government_eur_hab
+2015,AT1,2016,
+2017,AT1,2016,
```

Comment:
(2) the value seems to be missing here

Reply:
Data are missing at source

Reply:
In that case shouldn't we have a default like zero?